loadbalancer/lbalgorithm.go

```diff
@@ -0,0 +1,82 @@
+package loadbalancer
+
+import (
+	"math/rand"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+)
+
+type lbAlgorithmType int
+
+const (
+	RoundRobin         = "roundrobin"
+	unknownLbAlgorithm = "unknownLbAlgorithm"
+
+	roundrobin = iota
```

Comment:
this constant could be marked with `lbAlgorithmType` as

```
roundrobin lbAlgorithmType = iota
```

Reply:
done